### PR TITLE
doc(*): strike references to CoreOS

### PR DESCRIPTION
**Pull request by krancour**
Partially addresses #161

### CODE REVIEW

File: docs/src/installing-deis/system-requirements.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Deis components consume approximately 2 - 2.5GB of memory across the cluster, an
 * At least 4GB of RAM (more is better)
 * At least 40GB of hard disk space
 
-Note that these estimates are for Deis and CoreOS only, and there should be ample room for deployed applications.
+Note that these estimates are for Deis and Kubernetes only, and there should be ample room for deployed applications.
 
 Running smaller machines will likely result in increased system load and has been known to result in component failures, issues with etcd/fleet, and other problems.
```
Comment (arschles) [inline docs/src/installing-deis/system-requirements.md]: remove the fleet mention here

**Comment by arschles** [comment]
made one comment, but LGTM after the fix

File: docs/src/installing-deis/system-requirements.md (hunk 1 above)
Comment (krancour) [inline docs/src/installing-deis/system-requirements.md]: It's covered by another PR, but if I don't remove it, it leaves this looking odd.  So I'll fix that.

Comment (arschles) [inline docs/src/installing-deis/system-requirements.md]: ok, cool

**Pull request merged**